UKHomeOffice/cerberus-service
# Cop 9825

## Description
COP-9825: Bring back Selector matches to the task details

## To Test
Select any task on a task list page that has got a selector to see the selector on a Task Details page.

## Developer Checklist

\* Required

- [*] Up-to-date with main branch? \*

- [*] Does it have tests?

- [*] Pull request URL linked to JIRA ticket? \*

- [*] All reviewer comments resolved/answered? \*
